### ISSUE TITLE
No network connectivity with live ISO on Virtualbox 7.0.x

### ISSUE THREAD

**Issue by sirg** [Nitrux/nitrux-bug-tracker#181]
# Issue Description

Unable to get any network connectivity from live ISO under Linux Mint/Virtualbox 7.0.18.
Similar results reported with Windows/VB 7.0.10.
Similar behavior with NAT and Bridged Adapter.
But no issues with KVM.

Please make sure to include a clear and concise description of what the bug is. 

**Steps to reproduce the bug**

1. Configure VirtualBox VM as generic or Debian  Linux/64 bit, 8GM memory, 20GB disk, nitrux-nx-desktop-plasma-65624aeb-amd64.iso
2. Boot the VM
3. Observe multiple warnings and indications of lack of network connectivity

**Expected behavior**
Working networking by default, like with many other VMs.

**Additional context**

Tried to turn off NetworkManager and manually config the link up with a static address. No response to ping from local router or the host. Exact same approach works fine on other VMs.

**OS version and Desktop information**
Mint 21.3, but same results seen on Win 10 with VirtualBox 7.0.10.

Open System Settings, go to About System, click "Copy to clipboard in English," and paste the content here.
Attached: about-nitrux.png
![about-nitrux](https://github.com/Nitrux/nitrux-bug-tracker/assets/1130104/68574375-496f-4b2c-af9c-c1c5ec88cb0e)

Please advise what specific additional info is needed and how to extract it w/o network connectivity. I guess I could configure a local virtual disk, save data to it and mount/read it on the host afterwards.
---

**Comment by UriHerrera** [comment]
Duplicate of #177

**Comment by sirg** [comment]
I do not believe this is a duplicate of #177: it's not a name resolution issue. The system fails to exhibit any connectivity with the host LAN - APR, ping, DHCP all fail.

**Comment by UriHerrera** [comment]
You're right, it isn't a duplicate of that issue. I booted the ISO in VBox 7.0.18 r162988, and networking works when using NAT and all of the adapters except `virtio-net` which may require adding modules to the initram and rebuilding it.

Using Bridged network does indeed not work with any of the adapters.

**Comment by sirg** [comment]
HTH, to get networking to work with NAT after trying bridged, I had to switch back-and-forth between adapter types.
Once you can ping something by IP, you still need to fix DNS per #177 to get full functionality (just commenting out the nameserver line in /etc/resolv.conf works)
